File: Workspace-Mayo/edu.clinicas/src/controladores/Inicio.cs
using System;
using Registros;
using Servicios;

namespace Controladores {

class Inicio {
        static IMenuInterfaz menuInterfaz = new MenuImplementacion();
        static IRegistroInterfaz registroInterfaz = new RegistroImplementacion();
        public static List<CitaDto> lista = [];
        static void Main(String[] args)
        {
            registroInterfaz.LeerArchivo();
            menuInterfaz.MostrarMenu();
        }
}

}

File: Workspace-Mayo-Java/edu.clinica.version.java/src/controladores/Inicio.java
package controladores;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

import dtos.CitaDto;
import servicios.MenuImplementacion;
import servicios.MenuInterfaz;
import servicios.RegistroInterfaz;
import servicios.RegistroImplementacion;

public class Inicio {

	
    public static List<CitaDto> lista = new ArrayList<>();
    public static Scanner entrada = new Scanner(System.in);
	
	public static void main(String[] args) {
		
		MenuInterfaz menuInterfaz = new MenuImplementacion();
	    RegistroInterfaz registroInterfaz = new RegistroImplementacion();
	    
	    registroInterfaz.LeerArchivo();
        menuInterfaz.MostrarMenu();
	    
	}

}
